build.gradle

```diff
@@ -408,6 +409,11 @@ dependencies {
     testImplementation 'org.springframework:spring-beans:5.3.20'
     testImplementation 'org.junit.jupiter:junit-jupiter:5.8.2'
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.2'
+    // Only osx-x86_64 and linux-x86_64 are available
+    if (osdetector.classifier in ["osx-x86_64", "linux-x86_64"]) {
+        testImplementation "io.netty:netty-tcnative-classes:2.0.54.Final"
+        testImplementation "io.netty:netty-tcnative-boringssl-static:2.0.54.Final:${osdetector.classifier}"
```

Comment:
OpenSSL will still be broken for windows for now, am I understanding this right?